system/include/emscripten/wasmfs.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#pragma once
+
+#include <stdint.h>
+#include <sys/stat.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct Backend* backend_t;
+
+// Obtains the backend_t of a specified path.
+backend_t wasmfs_backend_by_path(char* path);
```

Comment:
How about adding "get" to these names? `wasmfs_get_backend_by_path` etc. Otherwise the actual operation is not really stated.

Reply:
i thought it might've been a little too long. I can add it back in.